src/pages/Login.js

```diff
@@ -1,10 +1,16 @@
-import React, { useContext } from 'react'
+import React, { useContext, useState } from 'react'
 import { ThemeContext } from 'styled-components'
 import styled from 'styled-components'
 import Landing from '../containers/Landing'
 import { Button } from '../components/Input'
 import google from '../assets/icons/google.svg'
 import github from '../assets/icons/github.svg'
+import { useAuth, googleSignIn, githubSignIn } from '../utility/Auth'
+import { useLocation } from 'wouter'
+import { ErrorBanner } from '../components/ErrorBanner'
+
+const ErrorMessage =
+  'There was an issue logging you in. If this persists, please contact info@nwplus.io.'
```

Comment:
Can we do this, and perhaps put info@nwplus.io into constants and reuse it here?
```suggestion
import { A } from '../components/Typography'

const ErrorMessage = <>
  There was an issue logging you in. If this persists, please contact" 
  <A href="mailto:info@nwplus.io">info@nwplus.io</A>.
</>
```